webapp/components/wpt-runs.html

```diff
@@ -184,16 +185,26 @@
           }, {});
 
           // We flatten into an array of objects so Polymer can deal with them.
-          const firstRunTime = x => Object.values(x.runs)[0][0].created_at;
-          const flattened = Object.entries(testRunsBySha)
-            .map(({0: sha, 1: runs}) => ({ sha, runs }))
-            .sort((a, b) => firstRunTime(a) < firstRunTime(b) ? 1 : -1);
+          const firstRunTime = runs => {
```

Comment:
Maybe rename to `firstRunDate` to be more accurate, and `firstRunTime.getTime()` wouldn't look as redundant below.